solidity/contracts/Mailbox.sol

```diff
@@ -14,7 +15,7 @@ import {IMailbox} from "./interfaces/IMailbox.sol";
 import {Address} from "@openzeppelin/contracts/utils/Address.sol";
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
-contract Mailbox is IMailbox, Versioned, Ownable {
+contract Mailbox is IMailbox, Versioned, Ownable, Indexed {
```

Comment:
A bit pedantic but Indexed should be the either after Versioned or IMailbox and leave Ownable as the last item used in the linearization